spec/controllers/v2/profiles_controller_spec.rb

```diff
@@ -33,25 +33,8 @@
     context 'Authorized' do
       let(:rbac_allowed?) { true }
 
-      it 'returns base fields for each result' do
-        collection = items.map do |pf|
-          hash_including(
-            'id' => pf.id,
-            'type' => 'profile',
-            'attributes' => attributes.each_with_object({}) do |(key, value), obj|
-              obj[key.to_s] = pf.send(value)
-            end
-          )
-        end
-
-        get :index, params: { security_guide_id: security_guide.id, parents: %i[security_guide] }
-
-        expect(response).to have_http_status :ok
-        expect(response_body_data).to match_array(collection)
-
-        response_body_data.each do |pf|
-          expect(pf['attributes'].keys.count).to eq(attributes.keys.count)
-        end
+      it_behaves_like 'index', :security_guide do
+        let(:extra_params) { { security_guide_id: security_guide.id } }
       end
```

Comment:
If all these examples use the same `extra_params` in the same context, you can extract the `let` block to the same level to have it just once. This way you might be able to eliminate all the blocks on shared example calls ...

Reply:
If I extract the `let` blocks to a level above, the parameter `parents:` is not passed to the examples correctly, causing all tests to fail. Maybe there is a way to do this correctly, but I do not think this is it.